src/SFML/Graphics/Image.cpp

```diff
@@ -217,41 +211,32 @@ bool Image::loadFromMemory(const void* data, std::size_t size)
 
         if (ptr)
```

Comment:
Can we invert these conditions and reduce the nesting here since we're touching this function?

Reply:
I chose to not rearrange control flow in this PR for the sake of minimizing the patch size and making it as easy as possible to merge. I certainly prefer what you're proposing but don't want to go too far making unrelated style changes in this PR.

My goal to this end is still to enable clang-tidy [`readability-else-after-return`](https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html) which would force us to transform this code into something simpler. I'd love to get your support in enabling that clang-tidy check so we can make this transformation across the whole codebase and systematically enforce it in all future PRs so we no longer have to manually point this out in code review.

Reply:
I opened https://github.com/SFML/SFML/issues/3024 to discuss this.